### ISSUE TITLE
[a11y Support] Pension Team - Pension Ticket Review Sync

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#77913]
> [!IMPORTANT]  
> ## Ticket Connectivity
> - **Objective**: Looking into connectivity to other Pension Tickets.
> - **Details**: _The Pension Team changed, and there are tickets created with the old team, that may be able to be consolidated. Below are the remaining a11y-related tickets that need reviewed for connectivity for consolidation, review if need still exists and/or removal/closure._ 
> - **Tickets:** The tickets needing reviewed are listed below, with notes. 
> - **Next Steps Post Sync 3/8/2024:** Inquiring about a sync with Vicky re: outstanding Pension tickets and research status, will it include the new intake of Direct Deposit. Keeping backlog ticket and assigning Evan to the low priority 67037 for now, unless we hear it is not needed.
> - **Definition of Done**: See task lists in this ticket

### CAIA A11y Questions for Pension Team
- [ ] **Question 1**: When will research take place for Pension? Dates or timeframe needed. (See ticket #66842 ) and is it related to ticket `77464`?
- [ ] **Decision Q1:** Should we wait on research, place holds or close ticket? `keep open and wait` or `close ticket`
- [ ] **Question 2** Do any other tickets roll up to: #82734
- [ ] **Decision Q2:** Do we need to connect any more tickets in the task lists for the intake? `yes` or `no`
- [ ] **Question 3** new designs were done on 7/15/2024, is another review needed? Initial review was done in June by Jamie: #82734 and #84084
- [ ] **Related Item to Q3 above:** [Figma link for Midpoint July 15 2024](https://www.figma.com/design/9JKK5Eo43uJWEr66JPiebc/Pension-Claim-Form-21P-527EZ?node-id=5800-8169&t=V5dUy3B0nNzkKbAT-11)
- [ ] **Related Item to Q3 above:** [Currently implemented designs in Figma](https://www.figma.com/design/9JKK5Eo43uJWEr66JPiebc/Pension-Claim-Form-21P-527EZ?node-id=4750-62432&t=V5dUy3B0nNzkKbAT-11)
- [ ] **Decision Q3** Revisit and review new Figma designs `yes` or `no`

### Outreach
- [x] 3/8/2024 - Sara and Terry Synced
- [x] Reach out to Vicky [3/8/2024](https://github.com/department-of-veterans-affairs/va.gov-team/issues/66842#issuecomment-1985939647)
- [x] TBD - Sara, Terry and Vicky [reached out 3/8/2024](https://github.com/department-of-veterans-affairs/va.gov-team/issues/66842#issuecomment-1985939647) (Add Evan)
- [ ] **Sync with Vicky** on outstanding tickets for Pension, due to team shift and Laura S. `pending`
- [ ] Sync needed to review Open Sitewide Accessibility Tickets `12` Open as of 8/20/2024 and `16` Closed as of 8/20/2024
- [x] 8/1 and 8/2/2024 - Jamie reached out via the benefits-pension channel in Slack re: a11y needs for #82734
- [x] 8/26/2024 - Jamie reached out to follow-up in that same thread. See comments in this ticket for details.
- [ ] Feedback from Gretchen (last reached out 8/26/2024) - `Pending`

### Blocked Tickets
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/66842
- [ ] Plus potential Pilot(s) `no ticket for pilot yet`
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/82734

### Backlog Tickets
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/67037

### Icebox Tickets
- [ ] #48371
- [ ] **Related to Ticket 48371**: This is a ticket from 2022.  It was moved to the icebox by Randi. In July 2024, it was moved from the backlog to the icebox. Initial a11y recommendations were made by Josh Kim on 9/30/2021. Sharing this page with Maria to review, since a lot could have happened since then.

### Closed Tickets
- [x] **Revisit Ticket**: 67293 - for post-launch audit; assigned to Evan #67293
- [x] **Revisit Ticket**: #67293 - for post-launch audit; `assigned to Evan`
- [ ] #77900
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/84084

### Related Epics & Intakes for Pension
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/85381
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/66578
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/77464
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/82734

**Comment by sara-amanda** [comment]
## Ticket Status Update

### About this Ticket
This ticket has been updated to reflect the tickets for Pension that we are looking for next steps on. With the Pension team personnel change, we are trying to determine what tickets need to remain open and what work needs to be completed. 

### Next Steps

@NaomiPMC @strelichl thank you for helping CAIA in this effort!
- I just moved this ticket from `In Progress`, based on CAIA Hive Mind's new terminology for `Blocked`  👍 
- See the task lists in this ticket for questions and links to the tickets. 
- These are the remaining a11y-related tickets that need reviewed for connectivity, possible consolidation, and reviewed to see if the need still exists. 

cc: @EvanAtCoforma @coforma-jamie

**Comment by sara-amanda** [comment]
## Outreach Task List Updated
The **outreach task list in this ticket** has been updated to reflect the following:
- 8/1 and 8/2/2024, @coforma-jamie  reached out via the #benefits-pension channel, to inquire about a11y needs. 
- 8/26/2024, Jamie followed-up in that same thread. 
- As of 8/27/2024, waiting to hear back from Gretchen. 
- See comment in via the `blocked` intake [ticket #82734. ](https://github.com/department-of-veterans-affairs/va.gov-team/issues/82734#issuecomment-2312566056)

cc: @EvanAtCoforma @NaomiPMC @strelichl

**Comment by mistymg** [comment]
@lakper @coforma-jamie Is this a strictly a11y support ticket? It's still on our board and I was going to remove it. But wanted to check first.

**Comment by lakper** [comment]
@mistymg Yes, it can be removed. Thanks for checking!
